src/OrbitBase/Logging.cpp

```diff
@@ -5,15 +5,65 @@
 #include "OrbitBase/Logging.h"
 
 #include <array>
+#include <system_error>
 
 #include "absl/base/const_init.h"
 #include "absl/debugging/stacktrace.h"
 #include "absl/debugging/symbolize.h"
+#include "absl/strings/str_cat.h"
 #include "absl/strings/str_format.h"
 #include "absl/synchronization/mutex.h"
 
 static absl::Mutex log_file_mutex(absl::kConstInit);
 std::ofstream log_file;
+const int kLogFileLifetimeMinutes = 60 * 24 * 7;  // one week
+
+std::optional<int> GetLogFileLifetimeMinutes(std::string log_file_name) {
```

Comment:
```suggestion
ErrorMessageOr<int> GetLogFileLifetimeMinutes(std::string log_file_name) {
```
I think it would valuable if we see in the logs if this filename parsing fails. Currently if the parsing fails, its just silently omitted.

Reply:
Thanks for the suggestion! The parsing failures are also added to the log files now.